src/components/WebChat/WebChat.js

```diff
@@ -1,80 +1,95 @@
-import React, { useState, useEffect, useRef, useMemo } from "react";
+import React, { useState, useEffect, useRef } from "react";
 import { BsX, BsCursorFill } from "react-icons/bs";
 import { messageHistoryGenerator } from "./MessageGeneratorUtil";
 
-import "./WebChat.css";
-
 const history = {
   messages: messageHistoryGenerator(),
 };
 
 export default function WebChat() {
+ 
   const [chatHistory, setChatHistory] = useState(history.messages);
   const [isChatBoxVisible, setIsChatBoxVisible] = useState(false);
+  const [message, setMessage] = useState('');
   const chatBox = useRef(null);
 
   useEffect(() => {
-    setChatHistory(chatHistory);
-  }, [chatHistory]);
+    if(isChatBoxVisible)
+      chatBox.current.scrollTop = chatBox.current.scrollHeight;
+  }, [chatHistory, isChatBoxVisible]);
 
-  useEffect(
-    (event) => {
-      if (isChatBoxVisible)
-        chatBox.current.scrollTop = chatBox.current.scrollHeight;
-    },
-    [chatHistory, isChatBoxVisible]
-  );
+  const newMessage = () => {
```

Comment:
let's bring state management for webchat to the custom hook later.

Reply:
Sounds good